### ISSUE TITLE
Increment package and assembly versions in release/1.0.0

### ISSUE THREAD

**Issue by StephenBonikowsky** [dotnet/wcf#2849]
Currently the branch is still set to the versions last released...

System.Private.ServiceModel
- package: 4.1.1   update to 4.1.2
- assembly: 4.1.0.1   update to 4.1.0.2

System.ServiceModel.Duplex
- package: 4.0.2   update to 4.0.3
- assembly: 4.0.1.0   update to ?.?.?.?

System.ServiceModel.Http
- package: 4.1.1   update to 4.1.2
- assembly: 4.1.0.0   update to ?.?.?.?

System.ServiceModel.NetTcp
- package: 4.1.1   update to 4.1.2
- assembly: 4.1.0.0   update to ?.?.?.?

System.ServiceModel.Primitives
- package: 4.1.1   update to 4.1.2
- assembly: 4.1.0.0   update to ?.?.?.?

System.ServiceModel.Security
- package: 4.0.2   update to 4.0.3
- assembly: 4.0.1.0   update to ?.?.?.?

**Comment by StephenBonikowsky** [comment]
In the servicing release last December (2017) we versioned forward all the packages but we only incremented the S.P.SM assembly leaving the 5 facades as they were.

For latest releases beginning with 2.1 we normalized the package and assembly versions and also decided that regardless of what the changes may be we will version all the packages and assemblies forward together.

Should we normalize the packages and assemblies in a similar fashion when/if we do another servicing release for 1.0.0?
